### PR TITLE
Define prop for setting copycat append batch size

### PR DESCRIPTION
**Pull request by ggezer**
Fixes https://github.com/Alluxio/alluxio/issues/9862

### CODE REVIEW

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -1289,6 +1289,13 @@ public String toString() {
                   + "(alluxio.master.embedded.journal.election.timeout).")
           .setDefaultValue("1s")
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE)
+          .setDescription(" Amount of data that is appended from leader to followers "
```
Comment (gpang) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: NIT: empty space prefix

also, what is the unit?

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 2]
```diff
@@ -1289,6 +1289,13 @@ public String toString() {
...
+              + "in single batch. Setting higher values might require increasing"
+              + "election timeout due to increased network delay.")
```
Comment (gpang) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: can you also mention the other tradeoff when the value is too low?

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 3]
```diff
@@ -1289,6 +1289,13 @@ public String toString() {
...
+          .setDefaultValue(1024 * 32)
```
Comment (gpang) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: this was the original default. Is there a better default?

Comment (ggezer) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: In a separate effort, I'll work on finding a sane default for this and other embedded journal properties.

File: core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (ggezer) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: unit is BYTES. Updated the name to reflect that.

**Comment by ggezer** [comment]
Jenkins, test this please

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 4]
```diff
@@ -3951,7 +3952,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.master.embedded.journal.addresses";
     public static final String MASTER_EMBEDDED_JOURNAL_ELECTION_TIMEOUT =
         "alluxio.master.embedded.journal.election.timeout";
-    public static final String MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE =
+    public static final String MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE_BYTES =
```
Comment (gpang) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: we can still keep the original name, so we can use abbreviations like `32KB`.

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 5]
```diff
@@ -1289,6 +1289,14 @@ public String toString() {
                   + "(alluxio.master.embedded.journal.election.timeout).")
           .setDefaultValue("1s")
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE_BYTES =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE_BYTES)
+          .setDescription("Amount of data that is appended from leader to followers "
+              + "in a single heartbeat. Setting higher values might require increasing"
```
Comment (gpang) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]:
```suggestion
              + "in a single heartbeat. Setting higher values might require increasing "
```

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 6]
```diff
@@ -1289,6 +1289,14 @@ public String toString() {
...
+              + "election timeout due to increased network delay. Setting lower values "
+              + "might stall knowledge propagation between the leader and followers.")
+          .setDefaultValue(1024 * 32)
```
Comment (gpang) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]:
```suggestion
          .setDefaultValue("32KB")
```

**Comment by ggezer** [comment]
alluxio-bot, merge this please

**Pull request merged**